newIDE/app/src/MainFrame/index.js

```diff
@@ -2266,6 +2266,7 @@ const MainFrame = (props: Props) => {
             openSubscriptionDialog(true);
           },
           project: state.currentProject,
+          saveProject,
```

Comment:
(see here the similar prop drilling we've done for "onChangeSubscription", that was called "onChangeSubscription" and not "changeSubscription". This is because a prop is called in *reaction* to something. It almost reads as english: "on change subscription requested, please close the export dialog and open the subscription dialog").